CtaReactiveMaster/Sources/Utilities/LoadingStatus.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  LoadingStatus.swift
+//  CtaReactiveMaster
+//
+//  Created by 伊藤凌也 on 2020/12/27.
+//
+
+import Foundation
+
+enum LoadingStatus {
+    case initial
+    case loading
+    case loadSuccess
+    case loadFailed
```

Comment:
```suggestion
    case loadFailed(Error)
```
の方がいいかもしれない errorハンドリングするかどうかは別として